plugins/vibin/skills/aurora-design-system/CHANGELOG.md

```
@@ -2,6 +2,12 @@
 
 All notable changes to this skill are recorded here. Format roughly follows [Keep a Changelog](https://keepachangelog.com/).
 
+## 2026-05-30
+
+### Added
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)